ynr/apps/uk_results/models.py

```diff
@@ -57,13 +57,22 @@ def as_dict(self):
             )
         return data
 
+    def versions_equal(self, v1, v2):
+        """
+        Remove the created key from the version dict
+        """
```

Comment:
This comment implies that we're changing one of the input, but we're just comparing them. It seems like a better description of this function would be more like "Compare v1 and v2, ignoring the created key"?

Reply:
Good catch, updated 

(comment was such because I was trying to compare the dict after changing it, but then realised that was a bad idea)